### ISSUE TITLE
RangeSlider messing up ranges

### ISSUE THREAD

**Issue by asonnleitner** [telerik/kendo-vue#257]
**Describe the bug**
When a quick range change is made, the range moves both thumbs.

**To Reproduce**

Drag one of the thumbs quickly to the other.

**Expected behavior**
The opposite thumb should not move.

**Additional context**
Video: https://drive.google.com/file/d/1hz4BN09JqxEAFG_Ack3oYDqn2DA-7IRc

**Comment by zdravkov** [comment]
Thank you for reporting the issue, @asonnleitner  We have successfully replicated it and will do our best to fix it as soon as possible. 
As a side note - while we do monitor the issues submitted in this repository, sometimes we may miss some of them or accidentally delay answers for them. This is because in Kendo UI for Vue we have dedicated support channels for direct support questions, forums and feedback portal where we are able to provide the best support possible. So in case you have more questions we recommend using any of them - https://www.telerik.com/support/kendo-ui-for-vue.
Greetings,
Plamen
